pages/docs/frontend-modules/development.en-US.mdx

```diff
@@ -13,39 +15,120 @@ yarn
+Where the argument following `--sources` is the relative path to the frontend module you want to run.
+
+It is possible to run multiple frontend module simultaneously. For example, if you're making changes to the [Order Basket](https://zeroheight.com/23a080e38/p/71ad91-order-basket), you'll likely want to test those changes against the Medications and Orders frontend modules. To do that, you'd run:
 
 ```sh
-yarn start --sources 'packages/esm-patient-chart-app' --port=7000
+yarn start --sources packages/esm-patient-medications packages/esm-patient-orders-app
 ```
 
-And that should fire up a local dev server at `http://localhost:7000/openmrs/spa`.
+This should fire up a Webpack dev server running local copies of the Medications and Orders frontend modules. Any local changes should propagate to the dev server in your browser.
 
-When you fire up a dev server in this way, several things happen under the hood. If you look at the `scripts` section of a manifest file (`package.json`) in a frontend module, you should see a `start` script that looks something like:
+By default, the dev server will run on port 8080. If you wish to run the server on a different port, you can pass a port number as the argument to the `--port` flag. For example, to run a dev server for the Vitals frontend module on port 7000, you'd run:
 
-```json
-“start”: "openmrs develop"
+```sh
+yarn start --sources 'packages/esm-patient-vitals-app' --port=7000
 ```
 
-What this means is that running `yarn start` invokes the `develop` command of the `openmrs` tooling. This command starts a new frontend development session using the OpenMRS app shell. The tooling sets up the import map, adds entries for core applications from the app shell and overrides the entries for the modules running locally. Ultimately, the corollary is that you can make changes to frontend modules in your local environment and have those changes propagate to the dev server in your browser. The tooling allows you to do various other cool things, such as:
+<Callout type="info">
+  **`openmrs` tooling deep dive**
+
+`yarn start` is a shorthand for running the [openmrs CLI](https://github.com/openmrs/openmrs-esm-core/tree/main/packages/tooling/openmrs). More specifically, it's aliased to run `openmrs develop`, which creates a development build of the app shell and runs it against a given OpenMRS backend. It spins up a Webpack dev server on port 8080 and proxies requests to the default backend, which is hosted at https://dev3.openmrs.org. This dev server will by default run local copies of the frontend modules you specify and override their import map entries. All the other frontend modules run on that dev server should be the latest versions from the community dev3 server. With `openmrs develop`, you can also:
```

Comment:
Technically, I think the version of the app shell we run is the production build. Actually using a dev build would be a good idea... IIRC that's what the `debug` command does?